packages/open-payments/jest.config.js

```diff
@@ -13,5 +15,6 @@ module.exports = {
   modulePaths: [`<rootDir>/packages/${packageName}/src/`],
   id: packageName,
   displayName: packageName,
-  rootDir: '../..'
+  rootDir: '../..',
+  transformIgnorePatterns: [`node_modules/(?!.pnpm|${esModules.join('|')})`]
```

Comment:
transpile ESM ky package into CommonJS

Reply:
I think what will end up working is doing the same in Rafiki and by adding the Adonis library to the list of esmModules. Haven't yet to try, but it follows the same principle